doc/variableexplorer.rst

```diff
@@ -2,95 +2,213 @@
+Dictionaries
+~~~~~~~~~~~~
+
+When the type of a variable is a dictionary, double clicking it will show you a viewer displaying each one of the keys of the dictionary along with its associated value. You can double click any of the values to modify it. 
+
+.. image:: images/variable_explorer/variable-explorer-dictionary.png
    :align: center
    :alt: Dictionary editor displaying keys and their types, sizes, and values
 
 |
 
-.. image:: images/variable_explorer/variable_explorer_list_timedelta_edit.png
+~~~~~
+Lists
+~~~~~
+
+For variables that are of type list, the window pane only displays the first ten values of the list. If you want to see them all, double click the variable in the pane to open a viewer that will display the index, type, size and value of each element of the list. You can also modify any value by double clicking it in the viewer and writing the new one.
+
+.. image:: images/variable_explorer/variable-explorer-list.png
    :align: center
-   :alt: List editor displaying a list of timedeltas, showing one being edited
+   :alt: List editor displaying a list, showing one being edited
 
 |
 
-.. image:: images/variable_explorer/variable_explorer_array_2D_resize.png
+~~~~~~~~~~~~
+Numpy Arrays
+~~~~~~~~~~~~
+
+For Numpy Arrays, the window pane only shows some of the values of the array. You can open a viewer that shows you graphically in a "heat map" the values of the array, displaying each value in a colored box, depending on the size of the value. You can deactivate the background color by unchecking the option in the window. If the array is too large, background color will be automatically deactivated.
+
+.. image:: images/variable_explorer/variable-explorer-array-2D-resize.png
    :align: center
    :alt: Array editor with a 2D int array, displaying a "heatmap" of its values
 
 |
 
-|varexpcontextmenu| |histogram|
+If possible, you can also change the format of the values in the array, choosing the number of decimals that you want the array to display. For this, click the option format and in the window displayed you can set the desired format, using printf formatting. For more information go to "https://docs.python.org/3/library/stdtypes.html#printf-style-bytes-formatting".
 
-.. |varexpcontextmenu|
-   image:: images/variable_explorer/variable_explorer_inset_contextmenu_array.png
-   :width: 45%
-   :alt: Variable Explorer with a context menu, including plot/histogram options
 
-.. |histogram|
-   image:: images/variable_explorer/variable_explorer_histogram.png
-   :width: 50%
-   :alt: Plot window showing a histogram, generated via the previous options
+Additionally, you can adjust the size of the rows and columns of the array by moving the arrows. Clicking the option resize, will bring the chart back to its default size.
 
-|
+.. image:: images/variable_explorer/variable-explorer-array-resize.gif
+   :align: center
+   :alt: Array editor with a 2D int array, showing resizing of columns
+ 
+
+~~~~~~~~~~
+DataFrames
```

Comment:
The Dataframes section should be before the Numpy Arrays one because they are more important.

Reply:
On the basis of what? Dataframes are more specialized than arrays, and particularly in many disciplines arrays are much more commonly used than dataframes if the latter are even used at all. Furthermore, arrays are simpler and form the basis of dataframes, so they should naturally be discussed first.

Reply:
I agree with CAM on this one.

Reply:
Ok, no problem. It's really not critical and it'll avoid @juanis2112 to do some rewriting when moving these sections around too.